spec/unit/trackable_spec.rb

```diff
@@ -283,4 +284,33 @@ def my_changes
       end
     end
   end
+
+  describe '#tracker_class' do
+    class MyTrackerClass
```

Comment:
For a spec you don't want to declare classes like this because they are global (aka not scoped to this spec) and effectively pollute the rest of the specs. Instead do this:

```ruby
let(:my_tracker_class) { Class.new }
```